packages/scripts/src/commands/e2e-data/index.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+import { Command } from "commander";
+
+import parseCmd from "./parse";
+import { logProgramHelp } from "../../utils";
+
+const program = new Command("e2e-data").description("Parse E2E data");
```

Comment:
I think it makes sense to link up with the existing commands system so that we don't have to install all the added dependencies into the test-e2e repo (and will help linking in the future with existing parsing methods) - also really great to see you were able to see how this side of the codebase functions to make some sensible addition